tests/harness/cases/wkt_wrappers.proto

```diff
@@ -17,3 +17,6 @@ message WrapperUInt32 { google.protobuf.UInt32Value val = 1 [(validate.rules).ui
 message WrapperBool   { google.protobuf.BoolValue val = 1 [(validate.rules).bool.const = true]; }
 message WrapperString { google.protobuf.StringValue val = 1 [(validate.rules).string.suffix = "bar"]; }
 message WrapperBytes  { google.protobuf.BytesValue val = 1 [(validate.rules).bytes.min_len = 3]; }
+message WrapperRequiredString { google.protobuf.StringValue val = 1 [(validate.rules).string.const = "bar", (validate.rules).message.required = true]; }
+message WrapperRequiredEmptyString { google.protobuf.StringValue val = 1 [(validate.rules).string.const = "", (validate.rules).message.required = true]; }
```

Comment:
WrapperRequiredEmptyString isn't adding much beyond what's covered in WrapperRequiredString

Reply:
Discussed offline. This is helping to distinguish between the null and empty cases. So this test is helpful.